Configuration/Settings.yaml

```diff
@@ -8,3 +8,6 @@ TYPO3:
             'commandobject':
               dbType: 'blob'
               className: 'Netlogix\Cqrs\Doctrine\Type\CommandType'
+            'exceptionobject':
```

Comment:
Do we need another type for this or can we just reuse commandobject (maybe rename it to be more generic?)